ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -549,6 +549,9 @@ def around(a, decimals=0, out=None):
 def round(a, decimals=0, out=None):
     return ivy.round(a, decimals=decimals, out=out)
 
+@to_ivy_arrays_and_back
+def round_(a, decimals=0, out=None):
+    return ivy.round(a, decimals=decimals, out=out)
```

Comment:
Hey! I have tested the implementation locally and it seems to be failing for the `AssertionError`. Can you please resolve it?
```
E       AssertionError:  the results from backend paddle and ground truth framework jax do not match
E        32768!=48896 
```
```
E       AssertionError:  the results from backend tensorflow and ground truth framework jax do not match
E        -inf!=-1.0 
```
```
E       AssertionError:  the results from backend numpy and ground truth framework jax do not match
E        -inf!=-1.0 
```